merlin/models/tf/transforms/sequence.py

```diff
@@ -664,11 +705,17 @@ def call(
         targets : Union[tf.Tensor, tf.RaggedTensor, TabularData], optional
             The target values, from which the mask can be extracted
             if targets inputs._keras_mask is defined.
+        training : bool, optional
+            Flag that indicates whether in training mode, by default True
+        testing : bool, optional
+            Flag that indicates whether in evaluation mode, by default True
         Returns
         -------
         Union[tf.Tensor, tf.RaggedTensor]
             If training, returns a tensor with the masked inputs replaced by the dummy embedding
         """
+        if not training and not testing:
```

Comment:
Don't we need to replace the masked (selected) item with the trainable embedding even in inference mode, to avoid that issue you found in Transformers4Rec when you don't replace the input by the trainable embedding?

Reply:
You're right! I will update the logic to replace with the `mask` embeddings